melodic/distribution.yaml

```diff
@@ -2573,6 +2573,22 @@ repositories:
       url: https://github.com/RobotWebTools/interactive_marker_proxy.git
       version: master
     status: maintained
+  interactive_marker_twist_server:
+    doc:
+      type: git
+      url: https://github.com/ros-visualization/interactive_marker_twist_server.git
+      version: kinetic-devel
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/ros-gbp/interactive_marker_twist_server-release.git
+      version: 1.2.0-0
+    source:
+      test_pull_requests: true
```

Comment:
Did you follow the steps [here](http://wiki.ros.org/buildfarm/Pull%20request%20testing) to allow PR testing?

Reply:
Oh, probably not. Will remove.